packages/lit-dev-content/site/docs/v3/components/decorators.md

```diff
@@ -85,86 +84,122 @@ In the future when decorators become a native web platform feature, this may no
+
+{% aside "info" %}
+
+What does stage 3 mean?
+
+It means that the specification text is complete, and ready for browsers to implement. Once the specification has been implemented in multiple browsers, it can move to the final stage, stage 4, and be added to the ECMAScript standard. A stage 3 proposal will only change if critical issues are discovered during implementation.
+
+{% endaside %}
+
+### Earlier decorators
+
+Before the TC39 proposal reached stage 3, compilers implemented earlier versions of the decorator specification.
+
+Most notable of these is [TypeScript's "experimental decorators"](https://www.typescriptlang.org/docs/handbook/decorators.html) which Lit has supported since its inception and is our current recommendation for use.
+
+Babel has also supported different versions of the specification over time as can be seen from the [`"version"` option of the decorator plugin](https://babeljs.io/docs/babel-plugin-proposal-decorators#version). In the past, Lit 2 has supported the `"2018-09"` version for Babel users but that has now been dropped in favor of the "standard" `"2023-05"` version described below.
+
+### Standard decorators { #standard-decorators }
+
+"Standard decorators" is the version of decorators that has reached Stage 3 consensus at TC39, the body that defines ECMAScript/JavaScript.
+
+Standard decorators are supported in TypeScript and Babel, with native browser coming in the near future.
+
+The biggest difference between standard decorators and earlier versions of the decorators is that, for performance reasons, standard decorators cannot change the _kind_ of a class member – fields, accessors, and methods – being decorated and replaced, and will only produce the same kind of member.
+
+Since many Lit decorators generate accessors, this means that the decorators need to be applied to accessors, not class fields.
+
+To make this convenient, the standard decorator specification adds the `accessor` keyword to declare "auto-accessors":
+
+```ts
+class MyClass {
+  accessor foo = 42;
 }
 ```
 
-The `allowDeclareFields` setting is generally not needed, but it can be useful if you want to define a reactive property without using a decorator. For example,
+Auto-accessors create a getter and setter pair that read and write from a private field. Decorators can then wrap these getters and setters.
+
+Lit decorators that work on class fields with experimental decorators - such as `@property()`, `@state()`, `@query()`, etc. - must be applied to accessors or auto-accessors with standard decorators:
 
 ```ts
-static properties = { foo: {} };
+@customElement('my-element')
+export class MyElement extends LitElement {
 
-declare foo: string;
+  @property()
+  accessor greeting = 'Welcome';
 
-constructor() {
-  super();
-  this.foo = 'bar';
 }
 ```
 
-### Avoiding issues with class fields and decorators {#avoiding-issues-with-class-fields}
+#### Compiler output considerations
```

Comment:
I think this can still be a level-3 heading so it's a sibling to early and standard decorator sections, since it talks about both

Reply:
I was actually thinking if we might remove this all together and just add a link to the reactive properties doc's https://lit.dev/docs/components/properties/#avoiding-issues-with-class-fields next to where we talk about `useDefineForClassFields: false` for TS.

Babel decorators no longer have this issue because they _have_ to use `accessor` now.

We'd have to remove the circular link back to this section from the properties doc if we remove this.

Reply:
ohhh i was totally answer based on the diff. you meant the compiler output considerations. for that i agree.